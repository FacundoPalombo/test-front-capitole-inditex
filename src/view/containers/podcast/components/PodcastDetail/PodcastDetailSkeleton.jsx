import React from 'react';
import styles from './styles.module.scss';
import image from './asset.jpg';

const PodcastDetailSkeleton = () => {
  return (
    <aside className={styles.podcast_detail__card}>
      <img
        className={styles.podcast_detail__picture}
        src={image}
        alt="..."
        aria-label="Loading Podcast Detail"
      />
      <hr className={styles.podcast_detail__separator} />
      <h3 className={styles.podcast_detail__title}>...</h3>
      <em className={styles.podcast_detail__artist}>...</em>
      <hr className={styles.podcast_detail__separator} />
      <p className={styles.podcast_detail__description}>
        <strong>&nbsp;</strong>
        <i>&nbsp;</i>
      </p>
    </aside>
  );
};

export default PodcastDetailSkeleton;
