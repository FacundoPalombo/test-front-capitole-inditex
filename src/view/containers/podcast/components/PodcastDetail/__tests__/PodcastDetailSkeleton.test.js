import React from 'react';
import { render } from '@testing-library/react';
import PodcastDetailSkeleton from '../PodcastDetailSkeleton';

describe('PodcastDetailSkeleton unit test', () => {
  afterAll(jest.clearAllMocks);
  it('should render skeleton ok', () => {
    const { asFragment } = render(<PodcastDetailSkeleton />);
    expect(asFragment()).toMatchSnapshot();
  });
});
