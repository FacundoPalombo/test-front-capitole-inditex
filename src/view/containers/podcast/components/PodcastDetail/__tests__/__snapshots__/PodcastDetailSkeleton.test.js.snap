// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PodcastDetailSkeleton unit test should render skeleton ok 1`] = `
<DocumentFragment>
  <aside
    class="podcast_detail__card"
  >
    <img
      alt="..."
      aria-label="Loading Podcast Detail"
      class="podcast_detail__picture"
      src=""
    />
    <hr
      class="podcast_detail__separator"
    />
    <h3
      class="podcast_detail__title"
    >
      ...
    </h3>
    <em
      class="podcast_detail__artist"
    >
      ...
    </em>
    <hr
      class="podcast_detail__separator"
    />
    <p
      class="podcast_detail__description"
    >
      <strong>
         
      </strong>
      <i>
         
      </i>
    </p>
  </aside>
</DocumentFragment>
`;
